model/gcp/registry.yaml

```
@@ -195,3 +195,42 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.operation.config
+        stability: development
+        type: string
+        brief: >
+          Operation-specific request configuration options for Google generative AI operations.
+
+          The string value should be a valid JSON dictionary. The JSON dictionary should include
+          only an allowlisted subset of the request configuration that does not overlap with
+          configuration that has already been captured elsewhere through standard Semantic Conventions.
+        examples:
+         - '{"audioTimestamp":true}'
+         - '{"safetySettings":[{"threshold":"OFF"}]}'
+         - '{"thinkingConfig":{"includeThoughts":true},"routingConfig":{"autoMode":{"modelRoutingPreference":"BALANCED"}}}'
+        note: >
+          The data structure is dependent on the value of "gen_ai.operation.name" and should correspond
+          with the "*Config" data structure in the Google Gen AI SDK. The Google Gen AI SDK provides
+          these "*Config" data structures in multiple languages, generated from a common source of truth.
+          See "google.genai.types.*" in <https://github.com/googleapis/python-genai> and 
+          `com.google.genai.types.*` in <https://github.com/googleapis/java-genai> for examples.
+
+          Because the data structure is encoded in JSON format, the key names should be specified
+          using "camelCase". More generally, the names should align with the expectations of proto3
+          JSON encoding <https://protobuf.dev/programming-guides/proto3/#json>; for example, fields
+          that represent a timestamp in the local language should get mapped to the proto3 JSON
+          encoding of "google.protobuf.Timestamp" (i.e. RFC 3339 date format) per the specification
+          given in <https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp>. 
+
+          Instrumentations should not populate this attribute by default. Rather, instrumentations
+          should provide an opt-in mechanism through which the config information can be dumped
```

```suggestion
          SHOULD provide an opt-in mechanism through which the config information can be dumped
```

we usually use MAY so that instrumentations don't have to make anything configurable, but given it's a recommendation to Google-specific instrumentation, I think it's ok to use SHOULD here.

---

Done.